internal/api/tests/public/runsList_test.go

```diff
@@ -315,6 +315,29 @@ var _ = Describe("runsList", func() {
 		})
 	})
 
+	Describe("sparse fieldsets satellite", func() {
+		BeforeEach(func() {
+			run := test.NewRun(accountNumber())
+
+			playbookName := "sparse-playbookName"
+			run.PlaybookName = &playbookName
+			run.PlaybookRunUrl = "https://example.com/"
+
+			Expect(db().Create(&run).Error).ToNot(HaveOccurred())
+		})
+
+		It("returns playbook run name and url", func() {
+			raw := listRunsRaw("fields[data]", "playbook_name,playbook_run_url")
```

Comment:
So the API caller needs to specify `playbook_run_url` to get a field named `web_console_url` in the response? That's highly inconsistent. The field names should be made consistent - should also match the field names in the internal API.